api/controllers/admin.go

```diff
@@ -0,0 +1,69 @@
+package controllers
+
+import (
+	"os"
+	"strings"
+
+	"github.com/quintilesims/layer0/api/job"
+	"github.com/quintilesims/layer0/common/config"
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/zpatrick/fireball"
+)
+
+type AdminController struct {
+	JobStore job.Store
+}
+
+func NewAdminController(j job.Store) *AdminController {
+	return &AdminController{
+		JobStore: j,
+	}
+}
+
+func (a *AdminController) Routes() []*fireball.Route {
+	return []*fireball.Route{
+		{
+			Path: "/admin/config",
+			Handlers: fireball.Handlers{
+				"GET": a.GetConfig,
+			},
+		},
+		{
+			Path: "/admin/health",
+			Handlers: fireball.Handlers{
+				"GET": a.GetHealth,
+			},
+		},
+	}
+}
+
+func (a *AdminController) GetConfig(c *fireball.Context) (fireball.Response, error) {
+	publicSubnets := []string{}
+	privateSubnets := []string{}
+
+	VPCID := os.Getenv(config.ENVVAR_AWS_VPC)
```

Comment:
I'm not if this is breaking some rules unknown to me with the controller pattern we have, but I think it would be better to inject `config` into `*AdminController` --- then you can get these values a bit easier ala `a.Config.SomeValue()`